xtable-core/src/main/java/org/apache/xtable/avro/AvroSchemaConverter.java

```diff
@@ -132,8 +132,13 @@ private InternalSchema toInternalSchema(
           break;
         }
         if (schema.getType() == Schema.Type.FIXED) {
-          metadata.put(InternalSchema.MetadataKey.FIXED_BYTES_SIZE, schema.getFixedSize());
-          newDataType = InternalType.FIXED;
+          String xtableLogicalType = schema.getProp("xtableLogicalType");
```

Comment:
Can you put `xtableLogicalType` into a constant that we can reference in the other files as well?